galley/pkg/config/schema/resource/schema.go

```diff
@@ -74,6 +74,21 @@ type Schema interface {
 	Equal(other Schema) bool
 }
 
+type GroupVersionKind struct {
+	Group   string
+	Version string
+	Kind    string
+}
+
+var _ fmt.Stringer = GroupVersionKind{}
+
+func (g GroupVersionKind) String() string {
+	if g.Group == "" {
```

Comment:
This can not happen in istio

Reply:
yes it can, we read things from core v1 API group like pods